### PR TITLE
release 2.0.0alpha1

### PR DESCRIPTION
**Pull request by mbjones**
This PR is for an initial `2.0.0-alpha1` release which prepares the vegbank2 repository for our normal release cycle. It contains the following:

- Updated project structure and layout (#31)
- Contributing guidelines (#31)
- License files (#31)
- Initial work on database migration to newer Postgres version (#29)
- Initial work on Flyway database migration scripts (#29)
- Initial poetry build infrastructure for python package builds.

### CODE REVIEW

**Comment by mbjones** [comment]
Ping: @RWShelton @regetz I'd like to get this merged as a starting point to provide a better public face to the github repo and a starting point for incremental releases. Waiting on review from one of you.

**Comment by RWShelton** [comment]
Looks good to me, I've been working with the flyway stuff for a while and it's working well. My only question - I know it's a small commit, but does my last one with the baseline helm chart need to be reviewed individually before it goes in with this pull request? More of a process question for my own knowledge going forward than anything particular about this one.

**Comment by mbjones** [comment]
@RWShelton thanks. for your recent commits, in general and typically, each new major feature would indeed be in its own branch named after the feature it is implementing, and you would create a PR to pull that into develop. This process is described in detail in the [CONTRIBUTING.md](https://github.com/NCEAS/vegbank2/blob/develop/CONTRIBUTING.md) file. I think your current commit is fine though as a starting point in an alpha release.

I just added in the `tests` structure for pytest, so this branch now includes that as well. I will add a github action to run the tests in a sec. Then we can merge.

**Comment by mbjones** [comment]
Added GH Action to run `poetry run pytest` which is now passing. Can definitely be improved (it emits some warnings about doing a `pip install`, but its a start.

File: README.md
Comment (regetz) [inline README.md:21]: Do we want to keep the DataONE discussions link or is there something more appropriate?

File: CONTRIBUTING.md
Comment (regetz) [inline CONTRIBUTING.md:37]: I assume we want to link to https://github.com/NCEAS/vegbank2/issues

File: CONTRIBUTING.md
Comment (regetz) [inline CONTRIBUTING.md:18]: I assume we want to link to https://github.com/NCEAS/vegbank2/issues

File: CONTRIBUTING.md
Comment (regetz) [inline CONTRIBUTING.md:22]: Delete this bullet, and probably the next one unless we create a VegBank discussion board

File: CONTRIBUTING.md
Comment (regetz) [inline CONTRIBUTING.md:38]: https://github.com/NCEAS/vegbank2

File: CONTRIBUTING.md
Comment (regetz) [inline CONTRIBUTING.md:48]: https://github.com/NCEAS/vegbank2

File: README.md
Comment (regetz) [inline README.md:11]: Can/should we use an apporpriate VegBank email list?

**Review by regetz — APPROVED** [review]
I left a bunch of small comments where I saw especially DataONE and/or boilerplate links/etc that probably need to change, either as part of this PR or with the next one. Still marking as Approved to unblock this merge.

File: README.md
Comment (mbjones) [reply README.md:21]: That's a good question. I will leave this as is, but I opened a ticket (#34)  to update this and replace it with a vegbank-appropriate solution. The README is a template and needs a lot of work still.

File: CONTRIBUTING.md
Comment (mbjones) [reply CONTRIBUTING.md:37]: yes, fixed locally and I will push shortly.

File: CONTRIBUTING.md
Comment (mbjones) [reply CONTRIBUTING.md:18]: yes, will push shortly

File: CONTRIBUTING.md
Comment (mbjones) [reply CONTRIBUTING.md:22]: I generalized it rather than deleting it.

File: CONTRIBUTING.md
Comment (mbjones) [reply CONTRIBUTING.md:38]: Done.

File: CONTRIBUTING.md
Comment (mbjones) [reply CONTRIBUTING.md:48]: Done

File: README.md
Comment (mbjones) [reply README.md:11]: Changed to help@vegbank.org for now.

**Pull request merged**
